include/particles/particle.tcc

```diff
@@ -500,6 +500,24 @@ void mpm::Particle<Tdim>::map_mass_momentum_to_nodes() noexcept {
   }
 }
 
+//! Map nodal properties to nodes
+template <unsigned Tdim>
+void mpm::Particle<Tdim>::map_properties_to_nodes() noexcept {
```

Comment:
This says `map_properties` but we only map mass and momentum, we should reconsider naming